winrar.txt

```diff
@@ -0,0 +1,12 @@
+[Section]
+Name = WinRAR
+Version = 6.24
+License = EULA
+Description = WinRAR is a popular archive manager to compress or decompress files, it has its own format for compression without much loss (.rar) 
+Category = 12
+URLSite = https://www.winrar.es/
+URLDownload = https://d.winrar.es/d/52z1698348162/snDlprqnRvBfMkspFOJjgA/winrar-x32-624.exe
+SizeBytes = 3260000
```

Comment:
All your 4 added applications size in bytes is exactly dividable by 10000 without leaving a remainder? That is very unlikely please correct the file sizes for each application to have the exact amount of bytes!